--- disko-1.1.1/Makefile.orig	2008-03-05 16:37:46.000000000 +0100
+++ disko-1.1.1/Makefile	2008-09-04 07:01:49.208980604 +0200
@@ -37,8 +37,8 @@
 	@install -d "$(DESTDIR)$(libdir)/pkgconfig"
 	@echo "prefix=$(prefix)" > $(DESTDIR)$(libdir)/pkgconfig/$(PROJECTNAME).pc
 	@echo "exec_prefix=\$${prefix}" >> $(DESTDIR)$(libdir)/pkgconfig/$(PROJECTNAME).pc
-	@echo "libdir=\$${prefix}/lib" >> $(DESTDIR)$(libdir)/pkgconfig/$(PROJECTNAME).pc
-	@echo "includedir=\$${prefix}/inc" >> $(DESTDIR)$(libdir)/pkgconfig/$(PROJECTNAME).pc
+	@echo "libdir=$(libdir)" >> $(DESTDIR)$(libdir)/pkgconfig/$(PROJECTNAME).pc
+	@echo "includedir=$(incdir)" >> $(DESTDIR)$(libdir)/pkgconfig/$(PROJECTNAME).pc
 	@echo  >> $(DESTDIR)$(libdir)/pkgconfig/$(PROJECTNAME).pc
 	@echo "Name: $(PROJECTNAME)" >> $(DESTDIR)$(libdir)/pkgconfig/$(PROJECTNAME).pc
 	@echo "Description: Multimedia Framework on top of DirectFB" >> $(DESTDIR)$(libdir)/pkgconfig/$(PROJECTNAME).pc
